pandas/1.5.1/Dockerfile

```diff
@@ -0,0 +1,31 @@
+################## BASE IMAGE ######################
+
+FROM python:3.10.8
+
+################## METADATA ######################
+
+LABEL base_image="python:3.10.8"
+LABEL version="1"
+LABEL about.summary="Python library for data manipulation and analysis."
+LABEL software="pandas"
+LABEL software.version="1.5.1"
+LABEL about.tags="implemented-in::python"
+LABEL about.home="https://pandas.pydata.org/"
+LABEL about.software="https://pandas.pydata.org/about/index.html"
+LABEL about.documentation="https://pandas.pydata.org/docs/"
+LABEL about.license="BSD-3-Clause"
+LABEL about.licence_file="https://github.com/pandas-dev/pandas/blob/main/LICENSE"
```

Comment:
typo here about.license_file

Reply:
Thanks for catching that! Typo and extra metadata have been added